src/index.test.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+
+import Sidenav from './index'
+
+let props
+
+const dummyElement = (elValue) => React.createElement('li', { id: 'li1' }, elValue)
```

Comment:
```suggestion
const dummyElement = (elValue) => React.createElement('li', { id:`id${elValue}` }, elValue)
```
`id` is expected to be unique.